draft/protos/ydb_object_storage.proto

```diff
@@ -0,0 +1,36 @@
+syntax = "proto3";
+option cc_enable_arenas = true;
+
+package Ydb.ObjectStorage;
+option java_package = "tech.ydb.proto.draft.object_storage";
+option go_package = "github.com/ydb-platform/ydb-go-genproto/draft/protos/Ydb_ObjectStorage";
+
+import "protos/ydb_issue_message.proto";
+import "protos/ydb_value.proto";
+import "protos/ydb_status_codes.proto";
+
+message ListingRequest {
+  enum EMatchType {
```

Comment:
I can't find any usage of this enum. I can predict that it is a part of matching_filtet, but how to use it int TypedValue? List of possible types is fixed in ydb_value.proto

Reply:
Here is an example in Go:
```
{
	Items: []*Ydb.Value{
		{
			Value: &Ydb.Value_Uint32Value{
				Uint32Value: uint32(Ydb_ObjectStorage.ListingRequest_EQUAL),
			},
		},
	},
}
```